src/ASM/Integrand.h

```diff
@@ -153,7 +153,8 @@ class Integrand
     NODAL_ROTATIONS   = 64, //!< Integrand wants nodal rotation tensors
     XO_ELEMENTS      = 128, //!< Integrand is defined on extraordinary elements
     INTERFACE_TERMS  = 256, //!< Integrand has element interface terms
-    NORMAL_DERIVS    = 512  //!< Integrand p'th order normal derivatives
+    NORMAL_DERIVS    = 512, //!< Integrand p'th order normal derivatives
+    UPDATED_NODES    = 1024 //!< Integrand wants updated nodal coordinates
```

Comment:
for later, but we should change this to '1 << n' at some point imo.

Reply:
Instead of the enums? Not sure if I got it, but I kind of like the readability of the enum names instead of a 'cryptical' integer value, or is it possible to use << expressions in the enum-declaration above?

Reply:
Changed to 1<<n, for n=2,3,4,... seems to work fine